parser.temper

```diff
@@ -236,13 +238,13 @@ class Parser {
     let codeBuffer = new ListBuilder<String>();
     let popCodeBuffer(): Void {
       if (!codeBuffer.isEmpty) {
+        // TODO If we add `clear` to StringBuilder, we can use that here.
         seq.add(new CodePoints(codeBuffer.join("") { (it);; it }));
-        // TODO Add `clear` method to ListBuilder for efficiency.
-        codeBuffer.splice(0);
+        codeBuffer.clear();
```

Comment:
I've recently added `clear` to ListBuilder, but we don't yet have it in StringBuilder, so I left this loop on ListBuilder.